src/main/Main.js

```diff
@@ -228,22 +228,14 @@ async function discordHarvestUpdate({ results, harvests }) {
         const harvest = harvests[i];
         if (!harvest.harvestDecision) continue;
         if (value) {
-            const msg = [];
-            msg.push('```');
-            msg.push(`Strategy:    ${harvest.name}`);
-            msg.push(`Reinvested:  ${Util.displayBNasFloat(harvest.harvestable, 18)} PNG ($${Util.displayBNasFloat(harvest.gainUSD, 18)})`);
-            msg.push(`Transaction: ${value.transactionHash}`);
-            msg.push('```');
-            DiscordBot.sendMessage(msg.join("\n"), CONFIG.DISCORD.HARVESTS);
-            let embedObj = {
+            const embedObj = {
                 Color:'0x00aaff',
                 Title:`Strategy: ${harvest.name}`,
                 Thumbnail:Util.thumbnailLink(harvest.name),
-                URL:Util.cchainTransactionLink(value.transactionHash)
+                URL:Util.cchainTransactionLink(value.transactionHash),

```

Comment:
extra comma

Reply:
trailing commas have always been acceptable in arrays, and in objects since ECMAScript 5